pkg/installer/build.yml

```diff
@@ -0,0 +1,21 @@
+# linuxkit build template 
+#
+# Copyright (c) 2024 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+org: lfedge
+image: eve-installer
```

Comment:
Have we requested the dockerhub repo for its package to be created?

Reply:
Yes, request is open.